server/.docker/init_db.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 <<-EOSQL
+  DROP SCHEMA IF EXISTS beer_user CASCADE;
```

Comment:
@nflow Can you not the schema creation sql for this? Copy pasting the current schema into the docker init_db will break as soon as the schema is adapted.